### DISCUSSION TITLE
alias format “random characters”

### DISCUSSION THREAD

**Discussion by Nypheena** [simple-login/app#1699] [Feature Requests]
I’d like to suggest adding another alias format that has no dots as some providers such as epic games block domains based on it, random characters in 8-12 length, such as what anonaddy offers and Bitwarden does for its username generator, so like `28rfjw2d@simplelogin.com` which I personally also find prettier

**Comment by ghost** [comment]
I have repeatedly suggested this, but the developers stubbornly ignore me. It's like I'm writing into the void.

Well at least here @nguyenkims I hope to have the patience to write a response?

Here is my requests:
[One (ignored)](https://github.com/simple-login/app/discussions/1675) 
[Two (ignored too even with likes!)](https://github.com/simple-login/app/discussions/1480) 
[Three (random, but not alias)](https://github.com/simple-login/app/discussions/1466)

**Reply by Nypheena** [reply]
Damn that’s a shame, doubt I’ll have much better luck then lol, I recently moved to anonaddy because I prefer this format immensely and I’m using custom domain

**Comment by Nypheena** [comment]
@nguyenkims why the ignore? UUID is ugly and several sites block anything with dots because it’s the most common thing to randomize on an alias

**Reply by nguyenkims** [reply]
Hi do you have examples of websites blocking the dots? Dots are often used in email addresses and almost all popular email services support it.

**Reply by Nypheena** [reply]
epic games, they literally told me they blocked my email because dots aren't allowed as they are known to be disposable 

https://github.com/simple-login/app/pull/996 you even have this pull request you just need to finish it, having imgur.3afh81@simplelogin.com isn't pretty and also too identifying and 12946151-512515212-155925791-51982571 is waaaay too long it just is not pretty at all, this is the best happy medium to keep nothing identifying in the address and with anonaddy I haven't had it blocked including epic games

**Reply by nguyenkims** [reply]
Thanks, gmail, proton mail both support dot sign so I'm not sure that the reason given by epic games is really the case, maybe they have a list of disposable email domains and SL happens to be listed there.
About the pull request https://github.com/simple-login/app/pull/996/files, I'm going to ask our developers to take another look at it. From what I see, the pull request does 2 things at once: refactoring and adding new feature and usually we try to keep the pull request as small as possible.

**Reply by Nypheena** [reply]
Well you obviously don’t necessarily have to use that pull request you could do something yourselves and all it would take is a modified UUID version that cuts off everything after the first dash(-) 

gmail may support it but 99% of services use it for the same as you to add the randomized bit to not overlap with other users and epic knows that so they block it, even fastmail who has 100 domains that’s super obscure only worked once the dots(.) was removed 

As for proton supporting it that feels a lot more like security reasons than format support so that people can’t just add a dot(.) and impersonate an email address since if your email is protonsupportsdots@proton then it’ll accept emails on p.r.o.t.o.n.s.u.p.p.o.r.t.s.d.o.t.s@proton on the same address so

There's also services like aliexpress which blocks if they see their name in the email and while theres UUID and just random word generation the sort of filter that most other alias providers use thats just random characters about 8-10 characters is clean and fully random which is really nice

**Reply by Nypheena** [reply]
I know plex also has MAJOR issues with dots(.) and will cut off passwords and emails that has them, I don't know if they ever fixed that but when they had the major leak it was still a thing for me.

**Comment by ghost** [comment]
I made a new idea that would solve our problem but won’t create more pain in ass of our lazy devs ;)

Feel free to upvote it and comment

[Here it is](https://github.com/simple-login/app/discussions/1752)
